src/Credentials.js

```diff
@@ -22,6 +22,9 @@ export default class Credentials {
     if (params.requested) {
       payload.requested = params.requested
     }
+    if (params.verified) {
```

Comment:
@christianlundkvist the persons own credentials are not stored or created as JWT's. We need to handle them differently. Most use cases will not care about the JWT's and just want an easy way to get at their claims. The few cases that require JWT's need this.